src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -94,11 +94,11 @@ function start() {
   // on whether the unbundle RTCP checkbox is checked.
   var config = {'iceServers': iceServers, iceTransportPolicy: iceTransports};
   var pcConstraints = {};
-  var offerConstraints = {'mandatory': {'OfferToReceiveAudio': true}};
+  var offerConstraints = {offerToReceiveAudio: 1};
```

Comment:
For the sake of overall consistency, should this be named `offerOptions` as well?